sql/fixturesSQL_minimal.sql

```diff
@@ -48,7 +48,7 @@ CREATE TABLE IF NOT EXISTS `accounts` (
   `last_login_attempt` datetime DEFAULT NULL,
   `failed_login_attempts` int(11) NOT NULL,
   `blocked_reason` text,
-  `perm_role` enum('R_NOBODY','R_ETER','R_OUDLID','R_LID','R_BASF','R_MAALCIE','R_BESTUUR','R_PUBCIE','R_VLIEGER') NOT NULL,
+  `perm_role` enum('R_NOBODY','R_ETER','R_OUDLID','R_LID','R_BASF','R_MAALCIE','R_BESTUUR','R_PUBCIE','R_VLIEGER', 'R_SINEREGNO_BESTUUR') NOT NULL,
```

Comment:
Denk ik niet relevant, want we willen het liefst dit na deze week weer opruimen.